Binance.Net/Objects/BinanceTransfer.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Binance.Net.Converters;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using CryptoExchange.Net.Converters;
+
+namespace Binance.Net.Objects
+{
+    /// <summary>
+    /// The result of transfering from main to margin account or margin to main
+    /// </summary>
+    public class BinanceTransfer
+    {
+        /// <summary>
+        /// The order id as assigned by Binance
+        /// </summary>
+        public long TranId { get; set; }
```

Comment:
```suggestion
        [JsonProperty("TranId")]
        public long TransactionId { get; set; }
```
I like to use full names